tests/reporter/kubernetes.go

```diff
@@ -1104,6 +1106,11 @@ func (r *KubernetesReporter) executeContainerCommands(virtCli kubecli.KubevirtCl
 				failedExecuteCmdFmt,
 				command, target, stdout, stderr, err,
 			)
+
+			pod, err := virtCli.CoreV1().Pods(pod.ObjectMeta.Namespace).Get(context.Background(), pod.ObjectMeta.Name, metav1.GetOptions{})
+			if errors.IsNotFound(err) || (err == nil && (pod.Status.Phase != "Running" || !isContainerReady(pod.Status.ContainerStatuses, container))) {
```

Comment:
Did you tell me the error is not of kind `errors.IsNotFound`?

Why not just - `if err != nil && (pod.Status.Phase != "Running" || !isContainerReady(pod.Status.ContainerStatuses, container)))`?

Reply:
> Did you tell me the error is not of kind errors.IsNotFound?
it is not the same API call, now we are doing a client.Get, before that the error was the Execute on pod

about the 2nd question
if err != nil, you can't look on the result even, it is undefined / potential not exists at all, anyhow the result is not valid as the Get failed (hence err != nil)